packages/client/index.js

```diff
@@ -56,7 +56,7 @@ function computeURLWithoutPath (url) {
   return url.toString()
 }
 
-function buildCallFunction (baseUrl, path, method, methodMeta, operationId) {
```

Comment:
Always return the full-response, I'd say. What's wrong IN that?😉

Reply:
I could do but that would break the current API which returns just the parsed JSON response body by default.